index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile-Portofolio</title>
+    <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <nav>
+             <ul>
+               <li><a class="myLogo" href="">Abdi Bekele</a> </li>
+               <li><a href="" class="toggler-icon"><img src="./assets/images/humberger.png" alt="humberger-icon"></a></li>
+
+             </ul>
+    
+
+
+
+        </nav>
+      
+    </header>
+       
+           
+       
+    
+    <section class= "headline">
+
+        <div class= "heyThere">
+           
+            <h1>Hey There, </h1>
+            <h1>I'm Abdi</h1></div>
+        
+        <div class="career">I am a Software Developer</div>
+        <div class="description"><p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don’t hestiate to contact me.</p></div>
+    <div class="links">
+        <ul>
+    <li><a href="#"><img src="./assets/images/github.svg"></a></li>
+  
+
+    <li><a href="#" class ="link"><img src="./assets/images/link.png" ></i></a></li>
```

Comment:
Make sure please to stick as much as possible to the Figma file. You did a great job with the social media links. However, the color on the Linkin icon is different from the Figma file. Kindly change the color of the icon. :+1: :smiley: :smiley: 
![image](https://user-images.githubusercontent.com/98363075/192076931-abf56a20-2cc8-452c-8526-cf8d5c2de6a0.png)
![image](https://user-images.githubusercontent.com/98363075/192076938-b5df66eb-bc6f-45ef-9e9c-840ff1316d0a.png)